docs/serverless/02-01-serverless.md

```diff
@@ -12,19 +12,21 @@ Serverless relies on [Knative Serving](https://knative.dev/docs/serving/) for de
 
     >**NOTE:** Function Controller sets the Node.js 12 runtime by default.
 
-2. Function Controller (FC) detects a new Function CR.
+2. Before the Function can be saved or updated, it is verified sequentially by the [defaulting and validation webhooks](#supported-webhooks).
 
-3. FC creates a ConfigMap with the Function definition.
+3. Function Controller (FC) detects the new, validated Function CR.
 
-4. Based on the ConfigMap, FC creates a Kubernetes Job that triggers the creation of a Function image.
+4. FC creates a ConfigMap with the Function definition.
 
-5. The Job creates a Pod with the Docker image containing the Function definition. It also pushes the image to a Docker registry.
+5. Based on the ConfigMap, FC creates a Kubernetes Job that triggers the creation of a Function image.
 
-6. FC monitors the Job status. When the image creation finishes successfully, FC creates a Service CR (KService) that points to the Pod with the image.
+6. The Job creates a Pod with the Docker image containing the Function definition. It also pushes the image to a Docker registry.
```

Comment:
I didn't notice it before but the sentence `The Job creates a Pod with the Docker image containing the Function definition` indicates that job has already created an image with the definition of the Function itself and doesn't build this image. Maybe, we should change it to `The Job creates a Pod, which build Function's production Docker image` or something similar?